src/views/administration/vuln-sources/VulnSourceOSVAdvisories.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <b-card no-body :header="header">
+    <b-card-body>
+      <hr/>
```

Comment:
This reminds me: Is there a logo of OSV that we can include and place above the `<hr/>` here (see e.g. the GHSA page vs. the NVD page)? 

Not a PR blocker in any way, but it'd be a nice touch I think :)

Reply:
<img width="1440" alt="Screenshot 2022-07-21 at 5 06 39 p m" src="https://user-images.githubusercontent.com/5709882/180261575-4aec2ce1-8e5b-42be-bb9f-c4cca4ab6262.png">